utils/streams.js

```diff
@@ -0,0 +1,156 @@
+        .parse(process.argv);
+    executeTheAction();
+}
+function executeTheAction() {
+    if (process.argv.length < 3) {
+        commander.help();
+    } else if (process.argv[2] == '-h' || process.argv[2] == '--help') {
+        commander.help();       
+    } else {
+        if (!!commander.action) {
+            switch (commander.action) {
+                case 'reverse':
+                    reverse();
+                    break;
+                case 'transform':
+                    transform();
+                    break;
+                case 'outputFile':
+                    outputFile(commander.file);
+                    break;
+                case 'convertFromFile':
+                    convertFromFile(commander.file);
+                    break;
+                case 'convertToFile':
+                    convertToFile(commander.file);
+                    break;
+                case 'cssBundler':
+                    cssBundler(commander.path);
+                    break;
+                default:
+                    console.log('Err!! Invalid arguement passed. Please choose correct action!!!');
+                    break;
+            }
+        } else {
+            console.log('Err!! No argument passed. Please choose correct action!!!');
+            process.exit();
+        }
+    }
+}
+
+function reverse() {
+    process.stdin.on('data', function (data) {
+        var splitString = data.toString().split("").reverse().join("");
+        process.stdout.write(splitString);
+    });
+}
+
+//Transform stream to uppercase
+function transform() {
+    let transToUpperCase = function (data, encoding, next) {
+        this.push(data.toString().toUpperCase());
+        next();
+    }
+    process.stdin
+        .pipe(through(transToUpperCase))
+        .pipe(process.stdout);
+}
+
+function outputFile(filePath) {
+    if (!!filePath) {
+        console.log(filePath);
+        fs.createReadStream(filePath)
+            .on('data', (data) => {
+                data.toString();
+                process.stdout.write(data);
+                process.exit();
+            })
+            .on('error', () => {
+                console.error('Failed to fetch the file. Kindly check your file path!');
+            })
+    } else {
+        console.log('No file path was observed!!');
+        process.exit();
+    }
+}
+
+function convertFromFile(filePath) {
+    if (!!filePath) {
+        try {
+            process.stdout.write(convertCsvToJson(filePath));
+            process.exit();
+        } catch (err) {
+            console.error('Failed to fetch the file. Kindly check your file path!');
+        }
+    } else {
+        console.log('No file path was observed!!');
+        process.exit();
+    }
+}
+
+function convertToFile(filePath) {
+    if (!!filePath) {
+        try {
+            let writeStream = fs.createWriteStream('./' + filePath.substring(0, filePath.lastIndexOf('.')) + '.json');
```

Comment:
please use `const` when the variable won't be changed

Reply:
Updated code to use const